report/11_Systems-Design.md

```diff
@@ -1,19 +1,19 @@
 ### System Design 
-The MiniTwit application is a web app written in C# using ASP.NET. Data is stored using EntityFrameworkCore as a database abstraction layer working with a Postgres database. This stack was chosen since it is a well supported and maintained stack with plenty of available documentation. This means that many third-party packages are available, like SeriLog, which makes interaction with ElasticSearch painless, and Prometheus-net for plug and play communication with Prometheus.
+The new version of the MiniTwit application is a web app written in C# using ASP.NET. Data is stored using Entity Framework Core (EF Core) as a database abstraction layer working with a PostgreSQL database. This stack was chosen because it is both well supported and maintained, as well as well documented. Furthermore, this means that many third-party packages are available, like SeriLog which makes interaction with ElasticSearch painless and Prometheus-net for plug and play communication with Prometheus.
```

Comment:
I do not think we should call it the new version, since a current version will always be the new version.

Reply:
Also, the sentence "well supported and maintained, as well as well documented." needs fewer wells. Perhaps just "well supported, maintained, and documented."

Reply:
It was to create a distinction between the original Python version and our ASP.NET version. Do you have any ideas of how to accomplish this in a more clear way?